.github/workflows/full_test_deploy.yml

```diff
@@ -48,10 +48,33 @@ jobs:
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
       - name: Publish Image to ECR
         run: docker push $AM_CLEANUP_IMAGE_TAG
+  build_record_thumbnail_lambda:
```

Comment:
This issue occurred to me while I was working on this, and I created a [ticket](https://permanent.atlassian.net/browse/PER-9799) to at least turn the build steps into a reusable workflow. I think the environments + reusable workflows approach is the right one to take here, and we are actually using both of those features in some places already (the tests workflow is reusable, and we use environments to add protections to prod deploys). I think this might be a good project for me to undertake in the potholes sprint in the beginning of next month.